htdocs/wp-content/install.php

```diff
@@ -34,16 +34,19 @@ function wp_install( $blog_title, $user_name, $user_email, $public, $deprecated
   populate_options();
   populate_roles();
 
-  if ( $language ) {
+/*  if ( $language ) {
     update_option( 'WPLANG', $language );
   } else {
     update_option( 'WPLANG', 'fi' ); // Use fi as default if language is not defined
   }
+*/
+
+  update_option( 'WPLANG', 'en' ); // Use en until the sample content has been localized
 
   update_option('blogname', $blog_title);
   update_option('admin_email', $user_email);
   update_option('blog_public', $public);
-  update_option('blogdescription',__('Uusi WP-palvelu.fi -sivusto'));
+  update_option('blogdescription',__('A fresh WordPress installation by Seravo'));
```

Comment:
You could have here a if ( get_locale == 'fi' ) { etc structure instead of removing the line.